input/data/2014/2014A%26A...562A..40H/tev-000117.yaml

```diff
@@ -1,4 +1,23 @@
 source_id: 117
 reference_id: 2014A&A...562A..40H
 
-# TODO: add data
+data:
+  livetime: 120
+  significance: 8.2
+  excess: 117
+
+pos:
+  ra: {val: 18h18m4.8s, err: 3.1s, err_sys: }
+  dec: {val: -15d28m1s, err: 43s, err_sys: }
+
+morph:
+  type: gauss
+
+spec:
+  type: pl
+
+  norm: {val: 0.09, err: 0.02, err_sys: 0.02}
+  index: {val: 2.3, err: 0.3, err_sys: 0.2}
+
```

Comment:
A decorrelation energy of `e_ref = 1.9 TeV` is mentioned in the paper.
You have to add this, or the spectrum will be incorrect (i.e. `e_ref =  1 TeV` would be assumed)